go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/pingcap/tiflow
 
-go 1.23.12
+go 1.24.0
```

Comment:
the latest is go1.24.10

Reply:
Go 1.25 is already out for quite a while (latest: 1.25.4). Note that we can keep this line as is if we upgrade the CD images to 1.25.x.

Reply:
Who is in charge of the CD images? Is there a Dockerfile for that somewhere?

Reply:
@lance6716 I think this now moved to Go 1.25